templates.go

```diff
@@ -82,10 +82,14 @@ func NewCVPage(co *Company, in *CV) *Page {
 				s := Section()
 				// find start of oneliners
 				var split int
-				for i, p := range in.Experience {
-					split = i
-					if p.oneLiner {
-						break
+				if template_type == TemplateFull {
```

Comment:
I was considering refactoring parts of this code to new functions to make it clearer.

However, I figured it'd be a bit out of scope for a bugfix PR.